package.json

```diff
@@ -71,13 +72,16 @@
     "@dashevo/feature-flags-contract": "0.1.0",
     "ajv": "^7.2.4",
     "ajv-formats": "^2.1.0",
+    "bignumber.js": "^9.0.1",
```

Comment:
Do we use it?

Reply:
There is a warning from CBOR, that it needs it.

Reply:
I got it thanks